client/src/components/Jumbotron/Jumbotron.css

```diff
@@ -1,7 +1,10 @@
 .jumbotron {
-    height: 300px;
+    height: 400px;
     margin-top: 5%;
-}
+    background-image: url("https://source.unsplash.com/collection/1248393/1600x900");]
+    background-repeat: no-repeat;
```

Comment:
Not sure if this is working. Maybe add onto the class in the jumbotron.js file as this is tiling on a large monitor